toil/src/cgcloud/toil/toil_cluster.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from cgcloud.toil.toil_box import ToilBox, ToilLeader
+from cgcloud.mesos.mesos_cluster import CreateMesosCluster
+from subprocess import check_call
+
+log = logging.getLogger( __name__ )
+shared_dir="/home/ubuntu/shared/"
```

Comment:
The shared dir should live in ~mesosbox, not ~ubuntu. On the leader and the workers.